### PR TITLE
fix: add sidebar to private squad so it's uniform

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- In prep for the placeholders I decided to tackle this first else it be double work.
- We needed to add the sidebar to all squad post to make a uniform behaviour.
- Minus the share as it's pointless to share private squad posts

Screenshot 2024-03-15 at 11 42 43

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

AS-195 #done

### CODE REVIEW

**Review by capJavert — APPROVED** [review]
Looks good 👍

File: packages/shared/src/components/post/SquadPostContent.tsx [hunk 1]
```diff
@@ -88,26 +86,23 @@ function SquadPostContent({
...
+          'flex-1 flex-col tablet:flex-row',
         )}
         hasNavigation={hasNavigation}
       >
         <div
           className={classNames(
             'relative min-w-0 px-4 tablet:px-8',
             className?.content,
-            isPublicSquad && 'flex flex-1 flex-col',
+            'flex flex-1 flex-col',
```
Comment (omBratteng) [inline packages/shared/src/components/post/SquadPostContent.tsx:97]: Move this in with the other just above (`relative min-w...`)

File: packages/shared/src/components/post/SquadPostContent.tsx [hunk 2]
```diff
@@ -88,26 +86,23 @@ function SquadPostContent({
         className={classNames(
           'relative tablet:pb-0',
           className?.container,
-          isPublicSquad ? 'flex-1 flex-col tablet:flex-row' : '!pb-2',
+          'flex-1 flex-col tablet:flex-row',
```
Comment (omBratteng) [inline packages/shared/src/components/post/SquadPostContent.tsx:89]: Move this up into `relative tablet...`

File: packages/shared/src/components/post/SquadPostContent.tsx [hunk 3]
```diff
@@ -88,26 +86,23 @@ function SquadPostContent({
...
           <BasePostContent
             className={{
               ...className,
               onboarding: classNames('mb-6', className?.onboarding),
               navigation: {
-                actions: classNames(
-                  'ml-auto',
-                  isPublicSquad && 'tablet:hidden',
-                ),
+                actions: classNames('ml-auto', 'tablet:hidden'),
```
Comment (omBratteng) [inline packages/shared/src/components/post/SquadPostContent.tsx:105]:
```suggestion
                actions: 'ml-auto tablet:hidden',
```

**Review by omBratteng — CHANGES_REQUESTED** [review]
Looks good, just some minor tidying of classes

File: packages/shared/src/components/post/SquadPostContent.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/post/SquadPostContent.tsx:97]: Ah yes, I knew I wanted to do something still 😅
Thanks for the sharp eyes

**Pull request merged**